docs/fidesops/docs/ui/local_dev.md

```diff
@@ -24,6 +21,14 @@ From the root `fidesops` directory, run the following:
 
 This will navigate you to the `admin-ui` directory, and run the development environment.
 
-Visit `http://localhost:3000/` in your browser, and provide user credentials to log in. 
+Visit `http://localhost:3000/` in your browser, and provide your user credentials to log in. 
+
+## Authentication
+
+To enable stable authentication you must supply a `NEXTAUTH_SECRET` environment
+variable. The best way to do this is by creating a `.env.local` file, which Next
+will automatically pick up:
 
-!!! tip "While your [Root Account](#creating-the-root-user) can be used to access the UI, additional [endpoints](#managing-users) are available to create and manage individual users for production-grade deployments."
+```bash
+echo NEXTAUTH_SECRET=`openssl rand -base64 32` >> .env.local
```

Comment:
smart! this will definitely help users. I think 4-5 of us have ran into issue due to not running this command.